src/style.css

```
@@ -1 +1,41 @@
-/* add styles here */
+html{
+  margin: 0;
+}
+
+body{
+  margin: 0;
+}
+
+.stars{
+  display: flex;
+}
+
+.stars__star{
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) 50% no-repeat ;
+}
+
+.stars--1 :nth-child(-n +1){
+  background: url(images/star-active.svg) 50% no-repeat;
+}
+
+.stars--2 :nth-child(-n +2){
+  background: url(images/star-active.svg) 50% no-repeat;
+}
+
+.stars--3 :nth-child(-n +3){
+  background: url(images/star-active.svg) 50% no-repeat;
+}
+
+.stars--4 :nth-child(-n +4){
+  background: url(images/star-active.svg) 50% no-repeat;
+}
+
+.stars--5 :nth-child(-n +5){
+  background: url(images/star-active.svg) 50% no-repeat;
+}
```

try to merge it like
```suggestion
.stars--1 :nth-child(-n +1),
.stars--2 :nth-child(-n +2) {
  background: url(images/star-active.svg) 50% no-repeat;
}
```